Blockzilla/metrics.yaml

```diff
@@ -61,6 +61,20 @@ browser:
     no_lint:
       - BASELINE_PING
 
+  pdf_viewer_used:
+    type: counter
+    description: |
+      Counts the number of times pdf view is opened in browser.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-ios/issues/3525
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-ios/pull/3344
```

Comment:
Please be sure to update this to the correct data-review URL 🙏

Reply:
yes, thank you for the reminder. I will update it now!